index.html

```diff
@@ -0,0 +1,2620 @@
+                    ></path>
+                  </g>
+                </svg>
+              </a>
+            </div>
+            <div
+              class="btn-box d-flex align-items-center align-items-lg-end vertical-align"
+            >
+              <button
+                type="button"
+                class="btn p-0 pr-2 pl-2 mr-2 mb-3 text-light d-none d-sm-block"
+              >
+                <b class="login-text-icon">LOGIN</b>
+                <span class="login-text-logo"><i class="fas fa-user"></i></span>
+              </button>
+              <button
+                type="button"
+                class="btn p-0 pr-2 pl-2 mr-2 mb-3 rounded-pill d-flex align-items-center justify-content-center"
+              >
+                <b>SUBSCRIBE ></b>
+              </button>
+              <button
+                type="button"
+                class="btn d-flex d-lg-none align-items-center p-0 pl-1"
+              >
+                <svg
+                  width="2em"
+                  height="2em"
+                  viewBox="0 0 16 16"
+                  class="bi bi-list text-light"
+                  fill="currentColor"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    fill-rule="evenodd"
+                    d="M2.5 11.5A.5.5 0 0 1 3 11h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4A.5.5 0 0 1 3 7h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4A.5.5 0 0 1 3 3h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"
+                  />
+                </svg>
+              </button>
+            </div>
+          </div>
+
+          <!-- NAV BAR - menu row 1 -->
+
+          <nav
+            class="navbar-expand-lg navbar-light align-content-around border-top"
+          >
+            <ul
```

Comment:
- [ ] The nav bar elements should be aligned center in the y-axis the same way they are in the original site.

For example, how your md nav looks:
![Screenshot_2021-01-29 Newsweek - News, Analysis, Politics, Business, Technology(2)](https://user-images.githubusercontent.com/16193212/106268815-61817080-620a-11eb-9fe0-5a579e8e85c3.png)


And how the original one looks like:
![header_mobile](https://user-images.githubusercontent.com/16193212/106268844-6d6d3280-620a-11eb-8e3c-07f35f54726b.png)

Reply:
done